config/master.key.bak

```diff
@@ -0,0 +1 @@
+626d146b0e45c9c4d740f5f7963eb5ba
```

Comment:
Do we need this file?

Reply:
No, added by accident - this isn't a key we ever use in any environment, was generated locally, I've removed now.